source/rate-limit.rst

```diff
@@ -32,6 +32,45 @@ With rate limiting you can collect profiling data for all database operations
 and reduce overhead by sampling queries.
 Slow queries ignore rate limiting and are always collected by the profiler.
 
+Comparing to "sampleRate" option
+================================
+
+The sampleRate option (= slowOpSampleRate_ config file option) is a similar
+concept to rateLimit. But it works at different profile level, completely 
+ignores ops faster than slowOpsThresholdMs (a.k.a. slowMs), and affects the 
+log file printing too. 
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * -
+     - sampleRate
+     - rateLimit
+   * - Affects profiling level 1
+     - yes
+     - no
+   * - Affects profiling level 2
+     - no
+     - yes
+   * - Discards/filters slow ops 
+     - yes
+     - no
+   * - Discards/filters fast ops 
+     - no
+     - yes
+   * - Affects log file 
+     - yes
+     - no
+   * - Example value of option
+     - 0.02
+     - 50
+     
+rateLimit is a better way to have continual profiling for monitoring or live
+analysis purposes. sampleRate requires setting slowOpsThresholdMs to zero if
+you wanted to sample all types of ops, and has an effect on the log file
+which may decrease or increase log volume.
```

Comment:
``rateLimit`` is a better way to have continuous profiling for monitoring or live
analysis purposes. ``sampleRate`` requires setting slowOpsThresholdMs to zero if
you want to sample all types of operations. It has an effect on the log file
which may either decrease or increase the log volume.

Reply:
The new "It" at the start of the last sentence becomes a bit unclear - is it rateLimit or sampleRate.

Suggest that "sampleRate" be repeated if ", and" is being removed.

Reply:
Agree